m4opt/missions/_ultrasat.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from regions import RectangleSkyRegion
+from synphot import Gaussian1D, SpectralElement
+
+from .. import skygrid
+from ..constraints import (
+    EarthLimbConstraint,
+    MoonSeparationConstraint,
+    SunSeparationConstraint,
+)
+from ..models import Detector
+from ..models.background import GalacticBackground, ZodiacalBackground
+from ..orbit import TLE
+from ..utils.dynamics import Slew
+from ._core import Mission
+
+ultrasat = Mission(
+    name="ultrasat",
+    fov=RectangleSkyRegion(
+        center=SkyCoord(0 * u.deg, 0 * u.deg), width=14.28 * u.deg, height=14.28 * u.deg
+    ),
+    constraints=[
+        EarthLimbConstraint(28 * u.deg),
+        SunSeparationConstraint(46 * u.deg),
+        MoonSeparationConstraint(23 * u.deg),
+        # GalacticLatitudeConstraint(10 * u.deg),
+    ],
+    detector=Detector(
+        # Total number of pixels
+        npix=89.9e6,
+        # Pixel scale, 5.4 arcsec / pixel
+        plate_scale=(5.4 * u.arcsec) ** 2,
+        # Area of an effective aperture of is 33 cm,
+        area=np.pi * np.square(0.5 * 33 * u.cm),
+        bandpasses={
+            "NUV": SpectralElement(
+                Gaussian1D,
+                amplitude=0.25,
+                mean=2600 * u.angstrom,
+                stddev=340 * u.angstrom,
+            ),
+        },
+        # Zodiacal light, Cerenkov radiation, and Stray light dominate ULTRASAT’s background noise.
+        background=GalacticBackground() + ZodiacalBackground(),
+        # Made up to match plot
+        read_noise=6,
+        dark_noise=12 / 300 * u.Hz,
+        gain=1,
+    ),
+    # ULTRASAT will be in a geosynchronous orbit similar to GOES-17.
+    # This is the TESS TLE downloaded from Celestrak at 2024-11-15T09:15:20Z.
+    # https://celestrak.org/NORAD/elements/weather.txt
```

Comment:
For which satellite is this TLE: GOES-7 or TESS?

Reply:
My bad. It's for the TEL : GEOS-17,  
https://github.com/nasa/dorado-scheduling/blob/main/dorado/scheduling/data/goes17.tle